setup.py

```diff
@@ -41,6 +41,7 @@ def get_version(path):
     url='https://github.com/tartley/colorama',
     license='BSD',
     packages=[NAME],
+    package_dir={"": "src"},
```

Comment:
Could I ask you to also fix line 43 for me, to resemble the one shown in Hynek's article? ie. from:

`packages=[NAME],`

to something like:

`packages=find_packages(where='src')'

where 'find_packages' should be imported from setuptools.

Yeesh. Look at that, we're alternatively trying to import distutils, too. We should perhaps drop that. You leave the distutils import alone, I'll go read about whether I can delete it. Sound good? Thoughts welcome.

Reply:
Maybe my suggestion is a bad idea after all. I'm terrified of breaking things for some project out there that is using distutils - if it turns out to be a high profile one, I'll get a tsunamic of issues, emails, tweets. Requires more thought. Ignore my suggestion above.